api/v1beta1/cinder_types.go

```diff
@@ -32,38 +32,20 @@ const (
 
 // CinderSpec defines the desired state of Cinder
 type CinderSpec struct {
-	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=cinder
-	// ServiceUser - optional username used for this service to register in cinder
-	ServiceUser string `json:"serviceUser"`
+	CinderTemplate `json:",inline"`
```

Comment:
+1 on composing the Spec, which is something I had in mind with the usage of `json:",inline"`.